README.md

```diff
@@ -1,70 +1,46 @@
-# Getting Started with Create React App
+# Math Magician
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> The website allows math lovers to play with numbers.
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- Major languages: JavaScript HTML & CSS 
+- linters
+- Frameworks: React
 
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
-
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Live Demo (if available)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+Getting Started
```

Comment:
- [OPTIONAL] Please it will be nice to update your `readme file` with explicit steps on how to clone and run your app locally, as this will be of help to non technical users.